packages/types-algosdk/index.d.ts

```diff
@@ -194,9 +194,18 @@ export interface MultiSig {
 
 export interface MultiSigAccount {
   // array of base32 encoded addresses
-  addrs: string[];
-  thr: number;
-  v: number;
+  address: string[];
```

Comment:
we can leave `addrs` here. Or change to `addresses`

Reply:
there are some inconsistencies in multisig address. why are we using two different value name ?
https://github.com/scale-it/algorand-builder/blob/2dcb8ba4f9bbaadeca2f5d3290c8b99844d14d38/packages/algob/src/lib/msig.ts#L98
here we are using `v` and `thr`
but in `account.ts` we are using version and threshold.
can you please elaborate ?
also if we use full names, wouln't multisig address example not work ?

Reply:
and in types-algosdk it is given as `version` and `threshold`

Reply:
One is multisignature Account and another is multisig (`msig` part of logic signature). 
![image](https://user-images.githubusercontent.com/33264364/99539582-6ccb4b00-29d4-11eb-8a02-f15aa80c366e.png)
We are using the full names(version, threshold) in account. We cannot change `v`, `thr` in msig. If you think this creates inconsistency then it's better to use `v` & `thr` everywhere. What you do think?

Reply:
I think we should keep it same as `v` and `thr`, and changes addres to `addresses`

Reply:
yes we can keep `v`, `thr` everywhere. For address list I would prefer `addrs`.

Reply:
`addresses` better depicts a list of `address`.
`addrs` can be anything address or addresses

Reply:
`algosdk` uses `addrs` for representing list of Algorand addresses everywhere. If you still want to keep it then fine, but i think we should use the same notations as much as we can.
![image](https://user-images.githubusercontent.com/33264364/99567386-1a018b80-29f4-11eb-91bd-a6efa995ae75.png)
![image](https://user-images.githubusercontent.com/33264364/99567422-284fa780-29f4-11eb-8332-3eedf38ce254.png)
![image](https://user-images.githubusercontent.com/33264364/99567604-59c87300-29f4-11eb-8cfc-b95273a821d3.png)

Reply:
ok then changing `v` to `version` and other parameters to same as given in `algosdk`

Reply:
Yes, let's use as in the SDK